roboticarm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1
    pid: {p: 0.5, i: 0.6, d: 0.2}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 1.0, i: 1.2, d: 0.4}
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint3
    pid: {p: 1.0, i: 1.0, d: 0.0}
  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint4
    pid: {p: 1.0, i: 1.0, d: 0.0}
  joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint5
    pid: {p: 1.0, i: 1.0, d: 0.0}
  joint6_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint6
    pid: {p: 1.0, i: 1.0, d: 0.5}
  joint7_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint7
    pid: {p: 1.0, i: 1.0, d: 0.5}
  joint8_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint8
    pid: {p: 1.0, i: 1.0, d: 0.5}
  joint9_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint9
    pid: {p: 1.0, i: 1.0, d: 0.5}